postcss.config.js

```diff
@@ -0,0 +1,15 @@
+import postcssPresetEnv from 'postcss-preset-env';
```

Comment:
I am not sure, but this file might not being used and needed based on the config [here](https://github.com/Larissa-Developers/prizy_landing/pull/10/files#diff-b9e136416b90437fa1dac910280b45fcR29)